### PR TITLE
Add Email validation

### PR DESCRIPTION
**Pull request by sntakirutimana72**
## In this PR, we:

- Added **email validation** JavaScript logic.
- Added custom error message display element in **html**.
- Fix linter errors.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -263,4 +263,19 @@ linksAction.forEach((node) => {
...
+submitForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const emailChecked = submitForm.elements[2];
+  const emailValue = emailChecked.value;
+  if (emailValue === emailValue.toLowerCase()) {
+    submitForm.submit();
+  } else {
+    submitForm.querySelector('.form-error')
+      .textContent = 'Email must be in lower case';
+  }
```
Comment (CollinsTatang) [inline js/index.js]: - [x] Please try to fix the email validation check, the user is still able to submit emails with both lower and uppercase letters.

- If a user submits the contact form with the email field containing uppercase characters an error message should show up.
- If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @MoRaad97, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 1 above)
Comment (MoRaad97) [reply js/index.js]: Hi @CollinsTatang,
Thanks for the review.
I and my partner @sntakirutimana72  checked the code, and It's working fine.
this ScreenShot of the project.
ScreenShot_20220513143340

**Comment by MoRaad97** [comment]
ScreenShot_20220513143340

File: js/index.js (hunk 1 above)
Comment (CollinsTatang) [reply js/index.js]: Please can we have a call?

Comment (MoRaad97) [reply js/index.js]: yes sure, I send you an invitation at zoom.

Comment (CollinsTatang) [reply js/index.js]: Ok

Comment (CollinsTatang) [reply js/index.js]: I have not received it please share here

Comment (MoRaad97) [reply js/index.js]: https://microverse.zoom.us/j/93398300976?pwd=TUVDSGRKbVkrQzlxWHdoL1N0cjRCZz09

Comment (CollinsTatang) [reply js/index.js]: All required changes have been solved.
The project is currently working as required on both ends.
Thanks for your time @MoRaad97

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁 
# Hello Team!😃 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [ ] As requested by the previous code reviewer, the Form is still accepting the email with uppercase and it doesn't show any error message. Kindly fix this. 👍🏻 
Screen Shot 2022-05-13 at 5 32 31 PM


_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 1 above)
Comment (MoRaad97) [reply js/index.js]: Thanks, @CollinsTatang 
It was very nice to meet you and chat.
And thanks for your advice.

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**